src/coreclr/vm/riscv64/asmconstants.h

```diff
@@ -255,7 +255,10 @@ ASMCONSTANTS_C_ASSERT(CallCountingStubData__TargetForThresholdReached == offseto
 #define PROFILE_LEAVE    2
 #define PROFILE_TAILCALL 4
 
-#define SIZEOF__PROFILE_PLATFORM_SPECIFIC_DATA 312
+// NOTE: this should be 16-byte aligned as stack size.
+#define SIZEOF__PROFILE_PLATFORM_SPECIFIC_DATA  0x140
```

Comment:
Suggestion:
```C
ASMCONSTANTS_C_ASSERT(SIZEOF__PROFILE_PLATFORM_SPECIFIC_DATA % 16 == 0)
```

Reply:
Thanks very much.
Very good.
I will add it.